cf-flow-logs-template.yml

```diff
@@ -0,0 +1,45 @@
+AWSTemplateFormatVersion: 2010-09-09
+Parameters:
+    NetworkName:
+        Type: String
+        Description: Network name instances to be attached to
+        Default: Bison
+Resources:
+    VpcFlowLog:
+        Type: AWS::EC2::FlowLog
+        Properties:
+          DeliverLogsPermissionArn:
+            'Fn::GetAtt':
+                - IamRoleForFlowLogs
+                - Arn
+          LogGroupName: FlowLogsGroup
+          ResourceId: 
+            Fn::ImportValue:
+                !Sub ${NetworkName}Vpc
+          ResourceType: VPC
+          TrafficType: ALL
+          FlowLogRole:
```

Comment:
Incorrect indention)

Reply:
Fixed. Not sure when it became unindented and why it worked eventually 🙂